geopandas/array.py

```diff
@@ -280,6 +281,16 @@ def __init__(self, data, crs=None):
 
         self._crs = None
         self.crs = crs
+        self._sindex = None
+
+    @property
+    def sindex(self):
+        if self._sindex is not None:  # check for existing
```

Comment:
This could get trimmed slightly:

```suggestion
        if self._sindex is None:
            self._sindex = get_sindex_class()(self.data)
            
        return self._sindex
```